### PR TITLE
Update vendor files and fix WebXR detection

### PR DESCRIPTION
**Pull request by msub2**
This is the first in a series of several patches to restore A-Painter to a working state with the latest version of A-Frame. This updates some of the vendor files and their usage in index.html and silences some three.js warnings relating to addAtribute and setDynamic

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,11 +12,11 @@
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
     <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="vendor/aframe.min.js"></script>
```
Comment (dmarcos) [inline index.html:15]: I think we can remove the A-Frame on the vendor folder and use the CDN builds: https://aframe.io/releases/1.3.0/aframe.min.js

We were probably using a custom build of in-between releases.

Comment (msub2) [reply index.html:15]: Gotcha, I kinda suspected as much. Will change

File: index.html [hunk 2]
```diff
@@ -138,7 +138,7 @@
                   brush
                   if-no-vr-headset="visible: false"
```
Comment (dmarcos) [inline index.html:139]: I would probably change that logic. Hide the controllers by default and display when entering VR.

We might not need a component for just that and incorporate the logic in `paint-controls`:

```
// in init method.
this.onEnterVR = this.onEnterVR.bind(this);
this.el.sceneEl.addEventListener('enter-vr', this.onEnterVR);

// component method
onEnterVR: function () {
  this.el.object3D.visible = true;
}
```

File: src/sharedbuffergeometry.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ SharedBufferGeometry.prototype = {
 
     var mesh = new THREE.Mesh(geometry, this.material);
 
-    mesh.drawMode = this.primitiveMode;
+    //mesh.drawMode = this.primitiveMode;
```
Comment (dmarcos) [inline src/sharedbuffergeometry.js:65]: this one. not needed anymore?

File: src/components/if-no-vr-headset.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 var utils = AFRAME.utils;
 
 /**
- * Set properties if headset is not connected by checking getVRDisplays().
+ * Set properties if headset is not connected by checking isSessionSupported().
```
Comment (dmarcos) [inline src/components/if-no-vr-headset.js:5]: we can get rid of this component if we can incorporate the logic in `paint-controls` as described above

File: src/components/paint-controls.js [hunk 5]
```diff
@@ -18,6 +18,9 @@ AFRAME.registerComponent('paint-controls', {
     this.controller = null;
     this.modelLoaded = false;
 
+    this.onEnterVR = this.onEnterVR.bind(this);
```
Comment (dmarcos) [inline src/components/paint-controls.js:21]: I think we need by default:

this.el.object3D.visible = false;

Comment (msub2) [reply src/components/paint-controls.js:21]: Done

File: index.html [hunk 6]
```diff
@@ -12,11 +12,11 @@
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
     <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
     <script src="vendor/aframe-input-mapping-component.js"></script>
     <script src="build.js"></script>
     <script src="https://unpkg.com/aframe-gltf-exporter-component/dist/aframe-gltf-exporter-component.min.js"></script>
-    <script src="vendor/aframe-teleport-controls.min.js"></script>
+    <script src="vendor/aframe-blink-controls.min.js"></script>
```
Comment (dmarcos) [inline index.html:19]: Maybe we can also move to the CDN https://cdn.jsdelivr.net/npm/aframe-blink-controls@0.3.0/dist/aframe-blink-controls.min.js

**Comment by msub2** [comment]
Moved everything that I could to CDN. For whatever reason, the aframe-input-mapping-component in vendor/ doesn't match any version I can find on the repo or any CDN. Might have been a custom build from Fernando, but regardless it's probably best to keep this one for now because nothing else I tried worked correctly.

File: index.html [hunk 7]
```diff
@@ -11,13 +11,13 @@
       UPLOADCARE_PUBLIC_KEY = 'eb0df3191b88f403ad76';
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
-    <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
     <script src="vendor/aframe-input-mapping-component.js"></script>
     <script src="build.js"></script>
-    <script src="https://unpkg.com/aframe-gltf-exporter-component/dist/aframe-gltf-exporter-component.min.js"></script>
-    <script src="vendor/aframe-teleport-controls.min.js"></script>
```
Comment (dmarcos) [inline index.html:19]: Can we remove unnecessary files from the vendor directory?

Comment (msub2) [reply index.html:19]: Everything unnecessary has already been removed at this point I'm pretty sure. The only things still there are the input mapping component (still needed), OrbitControls (removed in a later PR), and ga+saveas, both of which are still used

File: index.html [hunk 8]
```diff
@@ -11,13 +11,13 @@
...
-    <script src="vendor/aframe-tooltip-component.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/clipboard@1.5.12/dist/clipboard.min.js"></script>
```
Comment (dmarcos) [inline index.html:17]: What is clipboard used for?

**Comment by dmarcos** [comment]
Cool thanks! Great start! Happy to see A-Painter getting love ❤️

File: index.html (hunk 8 above)
Comment (msub2) [reply index.html:17]: For the Copy URL button when you upload a painting

**Pull request merged**